SharpBucket/V2/Pocos/BranchingModelBranch.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace SharpBucket.V2.Pocos
+{
+    public class BranchingModelBranch
```

Comment:
After few tests an reading it seems to be effectively a Branch since it's the value returned by the type field by bitbucket

But since bitbucket returned only few props of the Branch type we may create a BranchInfo POCO that include just the properties returned by Bitbucket when Branch type is returned partially (with a ComitInfo type for the target property)

Using the Info suffix is the rule we try to apply everywhere in that case. Even if we still add few legacy exceptions like the UserShort and the NamedBranch types

Reply:
But using the Branch type and not creating a BranchInfo could be OK
Since bitbucket seems to offer capabilities to re-add filtered properties from partial responses:
https://developer.atlassian.com/bitbucket/api/2/reference/meta/partial-response

I never tried and this is clearly not yet supported by this client, but if we go in that direction we will drop the XxxInfo types to be ale to return additional types from partial responses...

Reply:
This was confusing not being familiar with the projects implementation. The `Branch` POCO simply contains the `name` property. However, looking at the other API is see that that simple POCO is used everywhere which drops all extraneous fields. As suggested I used the existing `Branch` type here.